server/backend/database/mongo/client.go

```
@@ -1073,16 +1075,52 @@
+
+		// create GridFS bucket
+		bucket, err := gridfs.NewBucket(db)
+		if err != nil {
+			return fmt.Errorf("failed to create GridFS bucket: %w", err)
+		}
+
+		uploadStream, err := bucket.OpenUploadStream(fmt.Sprintf("%s_snapshot", docRefKey.DocID))
+		if err != nil {
+			return fmt.Errorf("failed to open GridFS upload stream: %w", err)
+		}
+		defer uploadStream.Close()
```

golangci-lint (1.61.0)

Error return value of `uploadStream.Close` is not checked

(errcheck)